tests/test_nbr_health.py

```diff
@@ -6,6 +6,7 @@
 pytestmark = [
     pytest.mark.sanity_check(skip_sanity=True),
     pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('util') #special marker
```

Comment:
Not sure if the 'utility' marker falls under topology. Maybe we should have something like @pytest.mark.utility? @yxieca and @Azure/sonic-fundamentals, what do you think?

Reply:
This is ok for now.

Thinking about how we are going to use it. We want to run 'util' first, and particularly topology tests, e.g. 't0', 't1', 'any'.

'util' needs to be not in t0/t1/any.

Reply:
Yes. I suggest using 'util' cases in a separated command before t0/t1/any, like this:
```
py.test .... --topology util ...
py.test .... --topology t0  ...
py.test .... --topology any ...
```
Is this OK?